src/AnnotationDialog.js

```diff
@@ -225,14 +225,22 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      */
     hide() {
-        if (this.element && this.element.classList.contains(constants.CLASS_HIDDEN)) {
+        if (!this.element || (this.element && this.element.classList.contains(constants.CLASS_HIDDEN))) {
             return;
         }
 
         if (this.isMobile) {
             this.hideMobileDialog();
         }
 
+        const createSectionEl = this.element.querySelector(constants.SECTION_CREATE);
```

Comment:
seems like this code is duplicated in the `destroy` method, maybe turn it into a common method?